.pipelines/rp-params.yml

```diff
@@ -0,0 +1,40 @@
+parameters:
+  - name: gitCommit
+    type: string
+    default: c87a7be
+  - name: gitBranch
+    type: string
+    default: master
+  - name: location
+    type: string
+    default: eastus
+  - name: resourceGroup
+    type: string
+    default: $(UNIQUE_IDENTIFIER)-aro-${{ parameters.location }}
+  - name: databaseAccountName
+    type: string
+    default: $(UNIQUE_IDENTIFIER)-aro-${{ parameters.location }}
+  - name: keyvaultPrefix
+    type: string
+    default: $(UNIQUE_IDENTIFIER)-aro-${{ parameters.location }}
+  - name: dst_acr_name
+    type: string
+    default: $(UNIQUE_IDENTIFIER)aro
+  - name: aroImage
+    type: string
+    default: ${{ parameters.dst_acr_name }}.azurecr.io/aro:$(git rev-parse --short=7 HEAD) # do we have git?
```

Comment:
In theory yes, but in practice I came to realize that [VERSION](https://github.com/Azure/ARO-RP/blob/master/Makefile#L105C75-L105C82) isn't neccrially the commit. When the git commit also includes a git tag then [VERSION gets the git tag value instead of the git commit](https://github.com/Azure/ARO-RP/blob/master/Makefile#L27-L31), and this causes some issues when the aro image tag isn't aligned with the deployment version